### ISSUE TITLE
Shared monster messages

### ISSUE THREAD

**Issue by flambard-took** [mangband/mangband#1305]
Currently, players only receive messages about monsters who consider them closest.

It's been working fine (preventing message flood from your party members), yet, similarly to #785 some messages would've been helpful.

I'm thinking monsters picking up/crushing items, and other things that might concern everybody around.

**Comment by flambard-took** [comment]
Anyone wants to add some messages?

**Comment by igroglaz** [comment]
yep, it would make world more 'alive' if we would see what monsters do :)

**Comment by draconisPW** [comment]
Small bug: message for each player uses o_name which is the description of the object for the original player. It should use the description for each player to avoid for example "You see an orange potion. The monster picks up a potion of healing." if original player knew what a healing potion was but not the other player.
